src/conditional-flow/string-conditions.js

```diff
@@ -1,41 +1,73 @@
 // This function should return true if the passed string is equal to "Hello"
-function isHello (val1) {
-
+function isHello(val1) {
+  if (val1 === "Hello") {
+    return true;
+  } else {
+    return false;
+  }
   // TODO: write code in this function body to pass the tests
-
 }
 
 // This function should return true if the passed string is not equal to "Hello"
-function isNotHello (val1) {
-
+function isNotHello(val1) {
+  if (val1 !== "Hello") {
+    return true;
+  } else {
+    return false;
+  }
   // TODO: write code in this function body to pass the tests
-
 }
 
 // This function should return true if the string val1 is is longer
 // than string val2
-function isLongerThan (val1, val2) {
-
+function isLongerThan(val1, val2) {
+  if (val1.length > val2.length) {
+    return true;
+  } else {
+    return false;
+  }
   // TODO: write code in this function body to pass the tests
-
 }
 
 // This function should return true if the string passed in the function's first
 // argument has an odd number of vowels
 
-function hasOddNumberVowels (val1) {
+function hasOddNumberVowels(val1) {
+  let count = 0;
+  let smallVal = val1.toLowerCase();
+  for (i = 0; i < smallVal.length; i++) {
+    if (
+      smallVal[i] == "a" ||
+      smallVal[i] == "i" ||
+      smallVal[i] == "e" ||
+      smallVal[i] == "o" ||
+      smallVal[i] == "u"
+    ) {
+      count++;
+    }
+  }
+  if (count % 2 === 1) {
```

Comment:
This if statement isn't required as `count % 2 === 1` is a boolean statement, so you could just return it :)